spec/system/topics_unread_when_closed_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+describe "Topics unread when closed", type: :system do
+  before { SiteSetting.experimental_topic_bulk_actions_enabled_groups = "1" }
```

Comment:
```suggestion
  before { SiteSetting.experimental_topic_bulk_actions_enabled_groups = Group::AUTO_GROUPS[:admins] }
```

Reply:
Actually we don't need this line in the test, but I'll fix a similar one in another file in a follow up pr.